pkg/inst/tests/test_utils.R

```diff
@@ -35,3 +35,51 @@ test_that("reserialize on RDD", {
   
   unlink(fileName)
 })
+
+test_that("cleanClosure on R functions", {
+  y <- c(1, 2, 3)
+  g <- function(x) { x + 1 }
+  f <- function(x) { g(x) + y }
+  newF <- cleanClosure(f)
+  env <- environment(newF)
+  expect_equal(length(ls(env)), 2)  # y, g
+  actual <- get("y", envir = env)
+  expect_equal(actual, y)
+  actual <- get("g", envir = env)
+  expect_equal(actual, g)
+  
+  # Check for nested enclosures and package variables.
+  env2 <- new.env()
+  funcEnv <- new.env(parent = env2)
+  f <- function(x) { min(g(x) + y) }
```

Comment:
It's a little bit weird seeing `min()` applied to one argument - do we have a unary that we can use?